mod_reforged/hooks/config/strings.nut

```diff
@@ -1133,8 +1140,8 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Hits to the head with one-handed maces apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn] and if the target is already [dazed,|Skill+dazed_effect] apply [Stunned|Skill+stunned_effect] for one [turn.|Concept.Turn]",
-				"Hits to the head with two-handed maces apply the [Stunned|Skill+stunned_effect] for 1 [turn.|Concept.Turn]"
+				"Hits to the head with one-handed maces apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn] and if the target is already [Dazed,|Skill+dazed_effect] apply [stunned|Skill+stunned_effect] for one [turn.|Concept.Turn]",
+				"Hits to the head with two-handed maces apply [stunned|Skill+stunned_effect] for 1 [turn.|Concept.Turn]"
```

Comment:
Can change it to `"Hits to the head with two-handed maces [stun|Skill+stunned_effect] the target for 1 [turn.|Concept.Turn]"`

Reply:
Should the top line then be changed to:

"Hits to the head with one-handed maces [daze|Skill+dazed_effect] the target for 1 [turn|Concept.Turn] and if the target is already [dazed,|Skill+dazed_effect] [stun|Skill+stunned_effect] the target for one [turn|Concept.Turn] instead."

Reply:
Yes sounds good. You can make it more concise by removing the mention of `the target` after stun.